szyfrowanie-kwantowe.md

```diff
@@ -64,15 +189,77 @@ Roznice miedzy bitem a kubitem moznaby wytlumaczyc ponizszym porownaniem:
 
 ---
 
-# Podsumowanie:
+# Przykład użycia (QKD)
 
-Szyfrowanie kwantowe oferuje obiecującą ścieżkę do bezpieczeństwa informacji w erze komputacji kwantowej. Chociaż technologia ta jest wciąż w fazie rozwoju, jej zdolność do zapewnienia bezpieczeństwa komunikacji jest niezrównana w porównaniu z klasyczną kryptografią.
+Najbardziej znanym przykładem jest protokół **BB84**, opracowany przez Charlesa Bennetta i Gillesa Brassarda w 1984 roku.
+
+1. **Wybór Bazy i Stanów Kwantowych**:
+
+   - Nadawca (zwykle nazywany Alicją) generuje losowy ciąg bitów (np. $01011$).
+   - Dla każdego bitu Alicja losowo wybiera jedną z dwóch baz kwantowych:
+     - bazę prostokątną $(1)$
+     - bazę diagonalną $(2)$
+
+---
+
+# Przykład użycia (QKD) c.d.
+
+$$
+\begin{align}
+  0 \text{ jako } |0\rangle \\
+  1 \text{ jako } |1\rangle
+\end{align}
+\tag{1}
+$$
+
+$$
+\begin{align}
+  0 \text{ jako } |+\rangle &= \frac{|0\rangle + |1\rangle}{\sqrt{2}} \\
+  1 \text{ jako } |-\rangle &= \frac{|0\rangle - |1\rangle}{\sqrt{2}}
+\end{align}
+\tag{2}
+$$
+
+<!--
+@TODO
+
+Stan ∣+⟩ jest tzw. stanem superpozycji. Stany odpowiadaja odpowiednio polaryzacjom |0⟩ (wertykalny), |1⟩ (horyzontalny), |-⟩ (-45 stopni), |+⟩ (+45 stopni)
+
+Matematycznie, ∣+⟩ jest zapisywany jako średnia ważona - z równymi wagami - tych dwóch stanów
+
+(Tylko dla |+⟩ i |-⟩) Dzielnik sqrt_2 jest potrzebny do 'normalizacji' stanu, co oznacza, że całkowite prawdopodobieństwo znalezienia cząstki w jednym z tych stanów musi być równe 1. W mechanice kwantowej suma kwadratów amplitud - w tym przypadku wag - musi równać się 1.
+-->
+
+<!-- @TODO: Jacon, czy tlumaczyc matematyke za polaryzacja kwantow i pokazac przyklady polaryzacji swiatla czy lekki overkill? -->
```

Comment:
Imo overkill, ale jak masz czutkę żeby to dodać i o tym popowiadać to myślę że jak najbardziej może być xd

Reply:
Nieee jak mowisz ze overkill to overkill, wypierdole to